Modules/Course/app/Http/Controllers/CourseCategoryController.php

```
@@ -25,22 +27,32 @@ class CourseCategoryController extends Controller
+            ($validated['status'] ?? null) !== null,
+            fn ($q) => $q->where('status', $validated['status'])
         );
-        $orderBy = $request->order_by == 1 ? 'asc' : 'desc';
-        $categories = $request->get('par-page') == 'all' ?
+        $orderBy = ($validated['order_by'] ?? 0) == 1 ? 'asc' : 'desc';
+        $categories = ($validated['par-page'] ?? null) == 'all' ?
             $query->orderBy('id', $orderBy)->get() :
-            $query->orderBy('id', $orderBy)->paginate($request->get('par-page') ?? null)->withQueryString();
-        $parentCategory = CourseCategory::find($request->parent_id);
+            $query->orderBy('id', $orderBy)->paginate($validated['par-page'] ?? null)->withQueryString();
+        $parentCategory = CourseCategory::find($validated['parent_id'] ?? null);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$parentCategory'. (Unused Code Rules)

(UnusedLocalVariable)